src/storable/tests.rs

```diff
@@ -35,4 +35,29 @@ proptest! {
     fn f32_roundtrip(v in any::<f32>()) {
         prop_assert_eq!(v, Storable::from_bytes(v.to_bytes()));
     }
+
+    #[test]
+    fn optional_f64_roundtrip(v in any::<f64>()) {
+        let opt_v= Some(v);
+        prop_assert_eq!(Some(v), Storable::from_bytes(opt_v.to_bytes()));
+    }
```

Comment:
We never exercise the `None` case in these tests. Let's check this path as well:

```suggestion
    fn optional_f64_roundtrip(v in proptest::option::of(any::<f64>())) {
        prop_assert_eq!(v, Storable::from_bytes(v.to_bytes()));
    }
```